src/OperatorPrecedenceChange.php

```diff
@@ -35,7 +35,7 @@ public function getVersion(): string
         return $this->version;
     }
 
-    public function getNewPrecedence(): string
+    public function getNewPrecedence(): int
```

Comment:
The constructor accepts int, so the return type should be int to no ?
cc @fabpot since you're the one who made this class.

Reply:
indeed